### PR TITLE
Update Windows build readme to match current VS2017 and java installers

### PR DESCRIPTION
**Pull request by matthuisman**
## Description

Few minor changes I noticed today as building Kodi

## Types of change

- [ ] **Bug fix** (non-breaking change which fixes an issue)
- [ ] **Clean up** (non-breaking change which removes non-working, unmaintained functionality)
- [ ] **Improvement** (non-breaking change which improves existing functionality)
- [ ] **New feature** (non-breaking change which adds functionality)
- [ ] **Breaking change** (fix or feature that will cause existing functionality to change)
- [x] **Cosmetic change** (non-breaking change that doesn't touch code)
- [ ] **None of the above** (please explain below)

## Checklist:

- [ ] My code follows the **[Code Guidelines](https://github.com/xbmc/xbmc/blob/master/docs/CODE_GUIDELINES.md)** of this project 
- [ ] My change requires a change to the documentation, either Doxygen or wiki
- [x] I have updated the documentation accordingly
- [ ] I have read the **[Contributing](https://github.com/xbmc/xbmc/blob/master/docs/CONTRIBUTING.md)** document
- [ ] I have added tests to cover my change
- [ ] All new and existing tests passed

### CODE REVIEW

**Review by phunkyfish — APPROVED** [review]
Thanks @matthuisman

**Comment by matthuisman** [comment]
ooops, deleted this branch by accident but then restored

File: docs/README.Windows.md [hunk 1]
```diff
@@ -77,17 +77,17 @@ Default options are fine.
 Start the VS2017 installer and click `Individual components`.
 * Under **Compilers, build tools and runtimes** select
   * `Msbuild`
-  * `VC++ 2017 v141 toolset (x86,x64)`
+  * `VC++ 2017 version 15.x v14.x latest v141 tools`
   * `Visual C++ 2017 Redistributable Update`
-  * `Visual C++ compilers and libraries for ARM`
-  * `Visual C++ compilers and libraries for ARM64`
-  * `Visual C++ runtime for UWP`
+  * `Visual C++ compilers and libraries for ARM` (if compiling for ARM)
```
Comment (Paxxi) [inline docs/README.Windows.md:82]: There used to be a bug where visual studio complained and refused to open the projects unless arm tooling was installed.

Have you tested without it installed without getting any errors when loading a x86/x64 solution?

**Review by Paxxi — COMMENTED** [review]
Looks good to me except for the comment. If that is tested then go ahead and merge.

Thanks @matthuisman 😀

**Comment by Paxxi** [comment]
Did you close this by accident again @matthuisman ?

**Comment by phunkyfish** [comment]
I’m going to assume a yes 😉

File: docs/README.Windows.md (hunk 1 above)
Comment (Paxxi) [reply docs/README.Windows.md:82]: Ok I checked this and it's still an issue for UWP. the UWP "Workload" includes arm and arm64 compilers and if they're not installed Visual Studio refuses to load the projects.
```suggestion
  * `Visual C++ compilers and libraries for ARM` (if compiling for ARM or UWP)
```

File: docs/README.Windows.md [hunk 2]
```diff
@@ -77,17 +77,17 @@ Default options are fine.
...
+  * `Visual C++ compilers and libraries for ARM64` (if compiling for ARM64)
```
Comment (Paxxi) [inline docs/README.Windows.md:83]:
```suggestion
  * `Visual C++ compilers and libraries for ARM64` (if compiling for ARM64 or UWP)
```

**Comment by matthuisman** [comment]
After the comments - i wasnt actually sure if those arm libraries where needed or not.
So maybe we just keep them as is.

I never even opened VS. I just followed the build commands and did it in command prompt.
There isn't anything there about using the .sln to build

All I know at moment is I didn't need them for the below to make a working kodi.exe
cmake -G "Visual Studio 15 Win64" -T host=x64 %userprofile%\kodi
cmake --build . --config "Release"

I'll try run a BuildSetup.bat now without them and see if that completes

**Comment by Paxxi** [comment]
I think either remove it or add the suggestions I made to change it to (if not compiling arm or uwp). You should be able to accept the edits right on the page under the changes tab.

You don't have to test it, I  tested it myself yesterday

**Comment by matthuisman** [comment]
done :)

**Comment by Paxxi** [comment]
Thanks again 😀

**Pull request merged**